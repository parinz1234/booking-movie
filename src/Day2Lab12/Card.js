import React, { Component } from 'react'

class Card extends Component {
  state = {
    title: 'My Title',
    body: 'My Body',
    post: []
  }
  render () {
    <div id="card">
      <h1 id="title">{ this.state.title }</h1>
      <p id="body">{ this.state.body }</p>
    </div>
  }
}

export default Card