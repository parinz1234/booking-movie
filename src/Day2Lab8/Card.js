import React from 'react'

const Card = () => (
  <div>
    <h1 id="title">My Title</h1>
    <p id="body">body</p>
  </div>
)

export default Card