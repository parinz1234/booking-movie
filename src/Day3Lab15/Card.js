import React from 'react'
import styled from 'styled-components'

const Div = styled.div`
  padding: 20px;
`
Div.displayName = 'Div'

const Title = styled.h1`
  color: green;
  font-size: 20px;
`

const Content = styled.p`
  color: green;
`

const Card = () => (
  <Div>
    <Title>My Title</Title>
    <Content>My Body</Content>
  </Div>
)

export default Div