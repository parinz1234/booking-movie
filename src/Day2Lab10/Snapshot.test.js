import React from 'react'
import { shallow } from 'enzyme'

import Card from './Card'

it('Match its snapshot', () => {
  const props = {
    title: 'My Title',
    body: 'My Body'
  }
  const wrapper = shallow(<Card {...props} />)
  expect(wrapper).toMatchSnapshot()
})