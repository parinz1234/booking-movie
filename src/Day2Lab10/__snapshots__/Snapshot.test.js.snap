// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match its snapshot 1`] = `
<div>
  <h1
    id="title"
  >
    My Title 
  </h1>
  <p
    id="body"
  >
    My Body
  </p>
</div>
`;
